pkg/node/id.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package node
+
+import (
+	"crypto/rand"
+
+	base58 "github.com/jbenet/go-base58"
+)
+
+// ID is the unique identifier of a Node in the overlay network
+type ID string
+
+// String transforms the ID to a string type
+func (n *ID) String() string {
+	return string(*n)
+}
+
+// Bytes transforms the ID to type []byte
+func (n *ID) Bytes() []byte {
+	return []byte(*n)
+}
+
+// StringToID trsansforms a string to a ID
+func StringToID(s string) *ID {
```

Comment:
`IDFromString`, that way it's clearer it's a constructor-like and get's grouped nicer

Reply:
Check for invalid key

Reply:
changed the name but there's no point in checking for invalid keys because this is technically an invalid key. 
It was used only so we could make progress while the real ID work was being finished.

https://storjlabs.atlassian.net/browse/V3-429

is to replace this